### ISSUE TITLE
Wrong swap amount

### ISSUE THREAD

**Issue by neo2005th** [MetaMask/metamask-extension#15199]
**Describe the bug**
Wrong swap amount from converting PYR token to USDT, it  should be 2.5 times more

**Screenshots**
![image](https://user-images.githubusercontent.com/19604343/178514256-884286fe-89e0-42d1-9a26-d7fbe5fdd60d.png)
![image](https://user-images.githubusercontent.com/19604343/178515076-c3e81cad-ff3e-453c-a980-a638bc775672.png)
![image](https://user-images.githubusercontent.com/19604343/178515436-cef58634-5f93-438b-aa49-27278975fbc6.png)
![image](https://user-images.githubusercontent.com/19604343/178514690-ecb4b782-9678-4b09-b3f9-81f695f4e6b5.png)

**To Reproduce**
1. Go to 'MetaMask' wallet on Firefox extension
2. Click on 'Swap'
3. Select 'Swap from' PYR
4. Select 'Swap to' USDT
5. Click on Max
6. Click on 'Review Swap' button

**Expected behavior**
Shows the same USD amount for the swap, aprox. $4100

**Smartphone (please complete the following information):**
 - Device: [Windows NT]
 - OS: [Windows 11]
 - App Version [MetaMask 10.14.7]

-------------------------------------------------------------
_to be added after bug submission by internal support / PM_
**Severity**

- How critical is the impact of this bug on a user?
- Add stats if available on % of customers impacted
- Is this visible to all users?
- Is this tech debt?

**Comment by cloudonshore** [comment]
So on coingecko, the address for PYR is 0x430ef9263e76dae63c84292c3409d61c598e9682 https://www.coingecko.com/en/coins/vulcan-forged#markets

However, this token only has ~2000 holders https://etherscan.io/token/0x430ef9263e76dae63c84292c3409d61c598e9682
There is almost no dex liquidity for this token, trying to swap on 1inch gives nearly 100% price slippage https://app.1inch.io/#/1/swap/USDT/PYR

This token is deployed on both ethereum and polygon, and all of the dex liquidity appears to live on polygon: https://app.uniswap.org/#/swap?inputCurrency=0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270&outputCurrency=0x430ef9263e76dae63c84292c3409d61c598e9682&chain=polygon

Basically there’s no liquidity on ethereum dexex for this token, it’s all on polygon on uniswap
